src/schema/v2/artwork/__tests__/utilities.test.ts

```diff
@@ -0,0 +1,149 @@
+import { Artwork } from "types/runtime/gravity"
+import { isTooBig, isTwoDimensional } from "../utilities"
+
+describe("isTwoDimensional", () => {
+  let artwork: Artwork
+
+  beforeEach(() => {
+    artwork = {
+      ...artwork,
+      width: null,
+      height: null,
+      diameter: null,
+      depth: null,
+      metric: null,
+      width_cm: null,
+      height_cm: null,
+      depth_cm: null,
+      diameter_cm: null,
+    }
+  })
+
+  describe("artwork has diameter", () => {
+    it("returns false", () => {
```

Comment:
Just to confirm: a work in a 2D category doesn't _automatically_ get rejected if it has a diameter, right? 

(Many legit cases like that e.g. https://www.artsy.net/artwork/takashi-murakami-flower-parent-and-child-rum-pum-pum-2)

Reply:
Yes correct! We [skip](https://github.com/artsy/metaphysics/blob/c744241bdd4d59b4b0700716807e19144a10b4fd/src/schema/v2/artwork/index.ts#L617-L618) the `isTwoDimensional` check for works in one of our 2D categories

Reply:
aha, BUT: you did just make me realize there's something I missed. We still do the `isTooBig` check for 2D works, and that check currently fails unless a work has width and height. I'll add specs and update the code to check for width/height OR diameter 👍

Reply:
Addressed in https://github.com/artsy/metaphysics/pull/4042/commits/b9b54690f657f1adab17456c08f42d7056125f63